frontend/src/routes/user/login@user.svelte

```diff
@@ -20,49 +18,45 @@
 
 	async function login(): Promise<void> {
 		try {
-			await UserService.login({
-				requestBody: {
-					email: email,
-					password
-				}
-			})
+			const requestBody = {
+				email,
+				password
+			}
+
+			await UserService.login({ requestBody })
+			// Once logged in, we can fetch the workspaces
+			$usersWorkspaceStore = await WorkspaceService.listUserWorkspaces()
```

Comment:
I did not know that syntax was valid. Where did you find that :) ?

Reply:
See:
[](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_stores)

Behind the scenes Svelte has generated the code to declare the local variable $alert, subscribe to the alert store, update $alert whenever the store's content is modified, and unsubscribe when the component is unmounted. It will also generate the alert.set(...) statements whenever we assign a value to $alert.

It's a great way to avoid boilerplate code!